src/forcemove.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+#include "move.h"
+#include "energy.h"
+#include "units.h"
+
+namespace Faunus {
+
+namespace Move {
+
+/**
+ * @brief Base class for dynamics integrators
+ *
+ */
+class IntegratorBase {
+  protected:
+    Space &spc;
+    Energy::Energybase &energy;
+    IntegratorBase(Space &spc, Energy::Energybase &energy);
+    virtual ~IntegratorBase() = default;
+
+  public:
+    /**
+     * @brief Move particles by one time step and update their positions, velocities and acting forces
+     * @param velocities
+     * @param forces
+     */
+    virtual void step(PointVector &velocities, PointVector &forces) = 0; // todo shall we return real time progress?
+    virtual void from_json(const json &j) = 0;
+    virtual void to_json(json &j) const = 0;
+};
+
+void from_json(const json &j, IntegratorBase &i);
+void to_json(json &j, const IntegratorBase &i);
+
+/**
+ * @brief Symmetric Langevin velocity-Verlet method (BAOAB)
+ *
+ * The integrator can conduct normal velocity-Verlet, when gamma = 0.
+ */
+class LangevinVelocityVerlet : public IntegratorBase {
+    float dt;    //!< integration time step (picoseconds)
```

Comment:
- why float? everything else in faunus is double... 
- rename `dt` to `time_step` at least as the internal variable name

Reply:
Done & done. However, we should decide how `dt` and `gamma` shall be represented in json. I consider more descriptive names like `time_step` and `friction` or `friction_coef` better.

Reply:
I would also prefer more elaborate names, although Faunus input is generally not so. Here I prefer `time_step` and `friction`. Ping @SHervoe-Hansen

Reply:
I would personally opt for `time_step` and `friction_coef ` (most common) or `collision_feq` (second most common, but yields the most physical understanding of the parameter). I would avoid only using `friction` as keyword would give very little indication what it does if novice to the method.

Reply:
Without any solid linguistic backing, I would have thought it should be "coeff" (?). I would prefer not having abbreviations in the input and `friction` should be pretty telling. I suppose it shouldn't be confused with "coefficient of friction" which, if I understand it correctly, is a unit-less quantity.

Reply:
Honestly I think it would be fine to call it `friction` for keyword, the documentation will obviously have the full written names to explain what property it is!

Reply:
The keywords are renamed to `time_step` and `friction`.